packages/api/src/utils/parser.ts

```diff
@@ -16,6 +16,11 @@ import { ILike } from 'typeorm'
 import { v4 as uuid } from 'uuid'
 import addressparser from 'addressparser'
 import { preParseContent } from '@omnivore/content-handler'
+import {
+  findEmbeddedHighlight,
+  EmbeddedHighlightData,
+} from './highlightGenerator'
+import { HighlightData } from '../datalayer/highlight/model'
```

Comment:
Maybe we could use `import { Highlight } from '../../elastic/types'`

Reply:
Thanks, not even needed, just some junk VS Code accidentally added.